### ISSUE TITLE
2.5.0: Silent close on Windows 10 Pro x64 when opening media

### ISSUE THREAD

**Issue by m-p-3** [ozmartian/vidcutter#8]
I've been trying to use v2.5.0 x64 on Windows 10 Pro x64  and every time I try to load a video file, the program will silently terminate without any error message.

Does VidCutter creates some kind of log somewhere to help assist with what's going on?

Thanks,
m-p-3

**Comment by ozmartian** [comment]
Unfortunatly no logging as of the current version but its actually being added for the next release but we can still try a few things to get to the bottom of your Windows config issue. Sounds like you have a broken codec (Windows DirectShow is what Qt uses under Windows). Are you able to play/watch various videos without fail in regular video players? By that I dont mean VLC, which packs all codec support within itself, but (even though its a sucky player its good for benchmaring issues) give Windows Media Player a try with different video files to see if they all work just fine.

If anyone else reports the same outcome to me then I will take the time to look into it but as it stands right now it looks most likely to be a problem at your end. I've tested the installer and the app on Windows 10, 9 and even old Windows 2000 Server. All worked without fail, all had no previous install and were basically bare bones. This doesn't mean I am not supproting and going to try help you out though! :-)

First plan of attack is do your quick tests so I know the current state of media support at your end. Now  comes the technical investigation part of our journey :-)

Open a command prompt window and navigate to vidcutter's installation folder C:\Program Files\VidCutter. Run the vidcutter.exe from there so you can then see any error messages are something else useful. You will probably not see anything at all BUT if we set some environment variables up before running the EXE we can get some debug info out of the app and the pluging Qt is using.

`$ QT_DEBUG_PLUGINS=1 vidcutter.exe`

and

`$ GST_DEBUG=3 vidcutter.exe`   (change 3 to 4 for more verbose depth)

So basically including those variable assignment strings BEFORE the app EXE. The output from those will be overwhelming but just scroll through quckly looking for glaring error messages that should eaily stick out.

**Comment by m-p-3** [comment]
The video file I tested do open in Windows Media Player.

Trying to run these commands give the following message

`C:\Program Files\VidCutter>QT_DEGUB_PLUGINS=1 vidcutter.exe
'QT_DEGUB_PLUGINS' is not recognized as an internal or external command,
operable program or batch file.`

Now that you mention it, maybe the installer doesn't take into account any kind of missing dependencies. I'll try installing Qt on my machine and see if it changes anything.

**Comment by FelixINX** [comment]
I think he want to say `SET QT_DEBUG_PLUGINS=1 vidcutter.exe` and `GST_DEBUG=3 vidcutter.exe`, wich set user variables.

Also, when I launch vidcutter.exe from CMD, it dosen't stay active on the command line.

**Comment by ozmartian** [comment]
@FelixINX @m-p-3 

you dont need python installed for the Windows build.. its all prebundled....

can you try the latest v3 from the releases page... its a big update making it a much better app and i have tested it on a bare bones, newly installed Win8 and Win10 installation without fail so it should be all good for you now...

not sure why it wasn't before but lets see if latest build is okay before looking any further...

thanks!

**Comment by ozmartian** [comment]
FYI the new version also has a view logs option in the menu so you can check that out for debugging issues with problematic video files. BUT this doesn't help if you cannot get the app to launch at all obviously :-)

will try my best to sort you out if v3 is still not working, there are many alternative ways to install afterall but for Windows, the native built installer is best, all dependencies and python itself are bundled within so its just like any other standard Windows application...

**Comment by nozzlegear** [comment]
I was having this problem on Windows 10, looks like it's all working now with v3!

**Comment by FelixINX** [comment]
Working here. Thanks!
